features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/HostResourceSwRunMonitor.java

```diff
@@ -198,7 +197,7 @@ public void rowCompleted(SnmpRowResult result) {
                 final SnmpValue name = nameResults.get(nameInstance);
                 final SnmpValue value = statusResults.get(nameInstance);
                 // See if the service name is in the list of running services
-                if (name != null && value != null && match(serviceName, stripExtraQuotes(name.toString()))) {
+                if (name != null && value != null && match(serviceName, StringUtils.stripExtraQuotes(name.toString()))) {
```

Comment:
`stripExtraQuotes` and `StringUtils.stripExtraQuotes` aren't equivalent. This refactor changes the behavior.

This source appears to match what the `stripFrontBack` function actually does:
    https://github.com/xtuml/pt_antlr/blob/master/pt_antlr/antlr/StringUtils.java#L79